ovirtlago/reposetup.py

```diff
@@ -72,33 +73,55 @@ def wrapper(*args, **kwargs):
 def _fix_reposync_issues(reposync_out, repo_path):
     """
     Fix for the issue described at::
-        https://bugzilla.redhat.com/show_bug.cgi?id=1332441
+        https://bugzilla.redhat.com//show_bug.cgi?id=1399235
+        https://bugzilla.redhat.com//show_bug.cgi?id=1332441
+
     """
-    LOGGER.warn(
-        'Due to bug https://bugzilla.redhat.com/show_bug.cgi?id=1332441 '
-        'sometimes reposync fails to update some packages that have older '
-        'versions already downloaded, will remove those if any and retry'
+    if any([len(repo_path) == 0, len(reposync_out) == 0]):
+        LOGGER.warning(
```

Comment:
actually I did that by accident, but it turns out ``logging.warn`` is going to be deprecated, so I'll keep it that way. http://bugs.python.org/issue13235